bee-network/bee-autopeering/examples/node.rs

```diff
@@ -74,10 +72,10 @@ async fn main() {
 
     // Storage config.
     // No config is  necessary for the `InMemoryPeerStore`.
-    // let peer_store_config = ();
+    let peer_store_config = ();
 
     // Sled peer store:
-    let peer_store_config = SledPeerStoreConfig::new().path("./peerstore");
+    // let peer_store_config = SledPeerStoreConfig::new().path("./peerstore");
```

Comment:
Since it's the example I wanted to give the user an idea how to do it with a "real" database. IMO it adds value to the example, because otherwise, the user would have to consult the documentation first.

Reply:
I you don't mind so much, I'ld keep it for that reason.

Reply:
Alles gut

Reply:
I think I would write that intenion somewhere in the comments too, otherwise it's a little confusing initially (for me at least).